src/Service/VendorService/MusicBrainz/MusicBrainzVendorService.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * @file
+ * Service for updating data from 'Musicbrainz' tsv file.
+ */
+
+namespace App\Service\VendorService\MusicBrainz;
+
+use App\Exception\IllegalVendorServiceException;
+use App\Exception\UnknownVendorResourceFormatException;
+use App\Exception\UnknownVendorServiceException;
+use App\Service\VendorService\AbstractBaseVendorService;
+use App\Service\VendorService\ProgressBarTrait;
+use App\Utils\Message\VendorImportResultMessage;
+use Box\Spout\Common\Exception\IOException;
+use Box\Spout\Common\Exception\UnsupportedTypeException;
+use Box\Spout\Reader\Common\Creator\ReaderEntityFactory;
+use Box\Spout\Reader\ReaderFactory;
+use Box\Spout\Reader\CSV\Reader;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class MusicBrainzVendorService.
+ */
+class MusicBrainzVendorService extends AbstractBaseVendorService
+{
+    use ProgressBarTrait;
+
+    protected const VENDOR_ID = 9;
```

Comment:
Do we have an overview over vendor ID's allocations... as this will clash with my new upload service vendor :-) Guess this gets into the code base first :-)

Reply:
Nope. Have considered a better solution. Or a check to validate that we don't have duplicates.

Reply:
Perhaps it should be a string instead of an integer?